GLOB sdist-make: /workdir/blessed-1.20.0/setup.py
cpythonlibtest-unit-test-tests create: /workdir/blessed-1.20.0/.tox/cpythonlibtest-unit-test-tests
cpythonlibtest-unit-test-tests installdeps: mock, pytest, -rrequirements.txt
cpythonlibtest-unit-test-tests inst: /workdir/blessed-1.20.0/.tox/.tmp/package/1/blessed-1.20.0.zip
cpythonlibtest-unit-test-tests installed: blessed @ file:///workdir/blessed-1.20.0/.tox/.tmp/package/1/blessed-1.20.0.zip#sha256=8225dd59f1b3b8b0d5e6b81c7c1be478cccebf39e9258449985a43db2976e94d,exceptiongroup==1.1.3,iniconfig==2.0.0,mock==5.1.0,packaging==23.2,pluggy==1.3.0,pytest==7.4.3,six==1.16.0,tomli==2.0.1,wcwidth==0.2.10
cpythonlibtest-unit-test-tests run-test-pre: PYTHONHASHSEED='3679295275'
cpythonlibtest-unit-test-tests run-test: commands[0] | pytest -v --tb=native --junitxml /workdir/results/blessed/1.20.0/1/cpython-test-results.xml
ERROR: usage: pytest [options] [file_or_dir] [file_or_dir] [...]
pytest: error: unrecognized arguments: --cov --cov-append --cov-report=xml
  inifile: /workdir/blessed-1.20.0/tox.ini
  rootdir: /workdir/blessed-1.20.0

ERROR: InvocationError for command /workdir/blessed-1.20.0/.tox/cpythonlibtest-unit-test-tests/bin/pytest -v --tb=native --junitxml /workdir/results/blessed/1.20.0/1/cpython-test-results.xml (exited with code 4)
___________________________________ summary ____________________________________
ERROR:   cpythonlibtest-unit-test-tests: commands failed
